plugins/inputs/windows_event_log/wineventlog/wineventlog.go

```diff
@@ -347,18 +346,20 @@ func (l *windowsEventLog) getRecords(handles []EvtHandle) (records []*windowsEve
 			//old SSM agent Windows format
 			var recordMessage eventMessage
 			if err = xml.Unmarshal(descriptionBytes, &recordMessage); err != nil {
-				return nil, err
+				log.Printf("I! [windows_event_log] UTF16ToUTF8Bytes() err %v", err)
```

Comment:
Is this log message correct? It looks like a copy paste error

Reply:
No, good catch.
Will fix.